public/js/home.js

```diff
@@ -1264,6 +1264,31 @@ document.getElementById("default_open").click();
 
 //#ifndef lite
 $.ajax("/version").then(ver => $("#version").text(ver));
+
+$.ajax("/updates").then(upt => {
+	$("#updates").html(upt);
+	document.getElementById("changelog").outerHTML = "<h2 class='info-header'>Version History/What's New:</h2>";
+    var items = document.getElementById("updates").getElementsByTagName("h2");
+	for (var i = 0; i < items.length; i++) {
+	    items[i].className = "info-header";
+	}
+    //Hide Everything but first and second versions
+    for(var i = 3; i < items.length; i ++) {
+        items[i].style = "display: none";
+    }
+    items = document.getElementById("updates").getElementsByTagName("ul");
+    for(var i = 3; i < items.length; i ++) {
+        items[i].style = "display: none";
+    }
```

Comment:
This can also be simplified:
```suggestion
    const subpoints = document.querySelectorAll("#updates ul");
    subpoints.slice(3).forEach(x => { x.style.setProperty("display", "none") });
```